src/main/java/org/alliancegenome/curation_api/model/entities/AffectedGenomicModel.java

```diff
@@ -113,7 +113,7 @@ public class AffectedGenomicModel extends GenomicEntity {
 		"agmAssociationObject.secondaryIdentifiers",
 		"agmAssociationObject.secondaryIdentifiers_keyword"
 	})
-	@OneToMany(mappedBy = "agmAssociationObject", cascade = CascadeType.ALL, orphanRemoval = true)
+	@OneToMany(mappedBy = "agmAgmAssociationObject", cascade = CascadeType.ALL, orphanRemoval = true)
 	@JsonView({View.FieldsAndLists.class, View.AffectedGenomicModelDetailView.class})
 	private List<AgmAgmAssociation> agmAgmObjectAssociations;
```

Comment:
This should be `parental_populations` according to the LinkML model, and I don't think we need the `agmAgmAssociations` at all.

Reply:
This mapping is needed to retrieve all AgmAssociation records. Without this mapping we won't see those associations in which the AGM is on second position (agm-AGM).

Reply:
Is there a reason to have the double agmAgm in the name or not in the name. It should be one way or the other with these fields.

Reply:
@oblodgett Follows the paradigm of other associations - subject is shared amongst other AGM Associations, whereas object is specific to association so we have `agmAgmAssociationObject`, `agmAlleleAssociationObject`, etc.

Reply:
I see your point.  Should be:

Columns: agmassociationsubject_id, agmagmassociationobject_id
Indexes: agmagmassociation_agmassociationsubject_index, agmagmassociation_agmagmassociationobject_index
FKs: agmagmassociation_agmassociationsubject_id_fk, agmagmassociation_agmagmassociationobject_id_fk